File: code/code.js
/* YYY 2018-02-02 check 'use strict for ES' YYY */

// Simple JavaScript syntax highlight code
'use strict';

class SquareKlasse {
  constructor(valueIn) {
    this.valueIn = valueIn;
  }
  squaredFunktion() {
    return this.valueIn * this.valueIn;
  }
}

function SquareKlasseES4 (valueIn) {
  this.valueIn = valueIn;
}

SquareKlasseES4.prototype.squaredFunktion = function() {
  return this.valueIn * this.valueIn;
}

function calculateFunktion(xParam,yParam) {
  const vNice = Math.floor(Math.exp(yParam));
  let vOut = Math.floor(yParam);
  const vTop = vNice - (vOut + xParam);
  while (vOut < vTop) {
    vOut += 3;
  }
  const vObjekt = new SquareKlasse(vOut);
  return vObjekt.squaredFunktion() + vNice + xParam;
}

const BARKONST = 1;
const BAZKONST = 3.7;

var resultVar = calculateFunktion(BARKONST, BAZKONST);
console.log("The result is:",resultVar);
console.log("Are you sure",resultVar,"is the correct number?");

File: code/code.py
# Simple Python syntax highlight code
import math

class SquareKlasse(object): #illegal syntax!
    def __init__(self, value_in):
        self.value_in = value_in
    def squared_funktion(self):
        return self.value_in * self.value_in

def calculate_funktion(x_param, y_param):
    _v_nice = int(math.exp(y_param))
    _v_out = int(y_param)
    _v_top = _v_nice - (_v_out + x_param)
    while _v_out < _v_top:
        _v_out = _v_out + 3
    _v_objekt = SquareKlasse(_v_out)
    return _v_objekt.squared_funktion() + _v_nice + x_param

BAR_KONST = 1
BAZ_KONST = 3.7

result_var = calculate_funktion(BAR_KONST, BAZ_KONST)
print "The result is:", result_var
print "Are you sure %d is the correct number?" % result_var
